### DISCUSSION TITLE
Deploy Elasticsearch Cluster & Kibaba On Kubernetes - Vincze Janos Blog

### DISCUSSION THREAD

**Discussion by giscus** [jvincze84/jvincze84.github.io#8] [General]
# Deploy Elasticsearch Cluster & Kibaba On Kubernetes - Vincze Janos Blog

https://readthedocs.vinczejanos.info/Blog/2022/11/18/elk_on_kubernetes/

**Comment by nsgbegin** [comment]
Hi its quite resourceful
can u provide me how to deploy using helm chart and without using docker for early setup
thanks

**Comment by thechaak** [comment]
Link not working.
update it
